pom.xml

```diff
@@ -274,6 +241,36 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.jacoco</groupId>
+          <artifactId>jacoco-maven-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>jacoco-prepare-agent</id>
+              <goals>
+                <goal>prepare-agent</goal>
+              </goals>
+              <configuration>
+                <append>true</append>
+                <destFile>${jacoco.exec.file}</destFile>
+                <excludes>
+                  <exclude>*Lexer</exclude>
+                  <exclude>org.optaplanner.core.impl.testdata.domain.*</exclude>
+                  <exclude>*TestdataAbstractSolutionBasedSolution</exclude>
```

Comment:
Aren't these `src/test` classes? Why would they be included in code coverage in the first place?

Reply:
Added comments explaining why these classes need to be excluded.